recipe/patches/0003-compatibility-for-python-3.12.patch

```diff
@@ -0,0 +1,258 @@
+From 016bf20d8ffd2946b778f4ad21e0382377218618 Mon Sep 17 00:00:00 2001
+From: Sergey B Kirpichev <skirpichev@gmail.com>
+Date: Mon, 30 Oct 2023 07:04:00 +0300
```

Comment:
FWIW, this is getting the email and timezone from a commit that GH user made elsewhere, and then taking the time in that timezone of the comment where the patch was posted.